.release-please-manifest.json

```diff
@@ -1,13 +1,14 @@
 {
-  "packages/app-info": "2.3.0",
-  "packages/crash-handler": "3.0.9",
-  "packages/errors": "2.2.0",
-  "packages/eslint-config": "2.0.1",
-  "packages/fetch-error-handler": "0.1.2",
-  "packages/log-error": "3.1.7",
-  "packages/logger": "2.4.2",
-  "packages/middleware-log-errors": "3.0.9",
-  "packages/middleware-render-error-info": "4.1.1",
-  "packages/serialize-error": "2.2.1",
-  "packages/serialize-request": "2.2.1"
+	"packages/app-info": "2.3.0",
+	"packages/crash-handler": "3.0.9",
+	"packages/errors": "2.2.0",
+	"packages/eslint-config": "2.0.1",
+	"packages/fetch-error-handler": "0.1.2",
+	"packages/log-error": "3.1.7",
+	"packages/logger": "2.4.2",
+	"packages/middleware-log-errors": "3.0.9",
+	"packages/middleware-render-error-info": "4.1.1",
+	"packages/serialize-error": "2.2.1",
+	"packages/serialize-request": "2.2.1",
+	"packages/opentelemetry": "0.0.0"
```

Comment:
**nitpick:** the diff indicates that all lines in this file have changed, can we revert the whitespace so that the blame is maintained?

Reply:
@CyntiBinti the change is that all the two-space indentation has changed to tabs. This would normally be OK but Release Please will switch it back to two spaces so there'll be a lot of weird churn in this file. You can reduce the diff by finding all tabs and replacing them with two spaces.